x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -41,6 +41,7 @@ export const DEFAULT_SIGNALS_INDEX = '.siem-signals' as const;
 export const DEFAULT_PREVIEW_INDEX = '.preview.alerts-security.alerts' as const;
 export const DEFAULT_LISTS_INDEX = '.lists' as const;
 export const DEFAULT_ITEMS_INDEX = '.items' as const;
+export const DEFAULT_RISK_SCORE_PAGE_SIZE = 1000 as const;
```

Comment:
I'm not quite sure of a reasonable default here. This corresponds to the `size` of the composite agg response, so:

```
# identities / page_size = # of requests the Risk Score task will need to invoke
```

But the number of unique identities in a system is going to be quite variable, and I'm bad at guesstimates. Help?

Reply:
Additional context: this was previously written as a `terms` agg, with a size of [65k terms](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#search-aggregations-bucket-terms-aggregation-size). We decided against that route as it would limit the total number of identities that we could collect, but now we've got to decide on how to iterate through an undefined number of identities.